monkey/common/version.py

```diff
@@ -0,0 +1,25 @@
+# To get the version from shell, run `python ./version.py` (see `python ./version.py -h` for details).
+import argparse
+from pathlib import Path
+
+MAJOR = "1"
+MINOR = "8"
+PATCH = "0"
```

Comment:
Why are we not additionally the standard python package version variable?  That way we can easily also expose this as an inner variable.